example/test-env.json

```diff
@@ -0,0 +1,10 @@
+{
+    "values": [
+        {"key": "host", "value": "http://172.18.0.4:8080"},
```

Comment:
This needs to be determined somehow at runtime. Maybe just use an alias.

Reply:
have you tried just using http://mms:8080? if it's connected to the mms_default network the service name that's set in the compose file should route to the mms service

Reply:
Yes, I have tried using just mms and I get this error: getaddrinfo ENOTFOUND mms mms:8080